api/all/src/main/java/io/opentelemetry/api/internal/ImmutableSpanContext.java

```diff
@@ -51,6 +56,18 @@ static SpanContext create(
         /* valid= */ false);
   }
 
+  /**
+   * This method is provided as an optimization when {@code traceIdHex} and {@code spanIdHex} have
+   * already been validated. Only use this method if you are sure these have both been validated,
+   * e.g. when using {@code traceIdHex} from a parent {@link SpanContext} and {@code spanIdHex} from
+   * an {@code IdGenerator}.
+   */
+  public static SpanContext createBypassingValidation(
```

Comment:
Can you add a unit test, also demonstrating this passing with some input that fails with the other method? For some coverage + documentation scones